src/main/java/com/box/sdk/retention/RetentionPolicy.java

```diff
@@ -66,8 +104,28 @@ public RetentionPolicy(String policyName, RetentionPolicyType type, String reten
 		}
 	}
 
+	/**
+	 * Retrieve a plicy
+	 * @param api agent that makes calls against API
+	 * @param policyId ID of target policy
+	 * @return An exsting policy
+	 * @throws MalformedURLException
+	 */
+	public static RetentionPolicy.Info getRetentionPolicy(BoxAPIConnection api, String policyId) throws MalformedURLException {
+
+		BoxAPIRequest request = new BoxAPIRequest(api, new URL(api.getBaseURL() + RETENTION_POLICIES_URL_PATH + "/" + policyId), "GET");
+		BoxAPIResponse response = request.send();
+
+		try {
+			return new Gson().fromJson(IOUtils.toString(response.getBody(), "UTF-8"), RetentionPolicy.Info.class);
+		} catch (IOException e) {
+			e.printStackTrace();
+			return null;
```

Comment:
At some point in time we should revisit the exception handling and try to either use the SDK existing exceptions, or come up with our own if that makes sense. I'll open an issue for that so we don't forget.